python/paddle/fluid/framework.py

```diff
@@ -5136,3 +5138,58 @@ def device_guard(device=None):
     pre_device = switch_device(device)
     yield
     switch_device(pre_device)
+
+
+def set_flags(flags):
+    """
+    This function sets the GFlags value in Paddle.
+
+    Args:
+        flags (dict): A dict contains flags and it's value.
+
+    Examples:
+            .. code-block:: python
+
+                import paddle.fluid as fluid
+                fluid.set_flags({'FLAGS_FLAGS_eager_delete_tensor_gb': 1.0})
```

Comment:
FLAGS_FLAGS_eager_delete_tensor_gb ? Is it a typo? Double "FLAGS" at the beginning?

Reply:
Done.